source/upgrade/important-upgrade-notes.rst

```diff
@@ -22,7 +22,7 @@ Important Upgrade Notes
 |                                                    |                                                                                                                                                                  |
 |                                                    | SQL to check the presence of duplicate data:                                                                                                                     |
 |                                                    |                                                                                                                                                                  |
-|                                                    | ``SELECT user_id, board_id, count(*) AS count FROM focalboard_category_boards WHERE delete_at = 0 GROUP BY user_id, board_id HAVING count(*) > 1;``              |
+|                                                    | ``SELECT user_id, board_id, count(*) AS count FROM focalboard_category_boards GROUP BY user_id, board_id HAVING count(*) > 1;``                                  |
```

Comment:
Are you open to applying this same edit also for the changelog https://docs.mattermost.com/install/self-managed-changelog.html#important-upgrade-notes?

Reply:
BTDT - pr-6265